react/svg.native.js

```diff
@@ -0,0 +1,7 @@
+import Svg from 'react-native-svg';
```

Comment:
Style: In Gutenberg standards, this should be capitalized `SVG`

>Acronyms must be written with each of its composing letters capitalized. This is intended to reflect that each letter of the acronym is a proper word in its expanded form. If an abbreviation or an acronym occurs at the start of a variable name, it must be written to respect the camelcase naming rules covering the first letter of a variable or class definition. For variable assignment, this means writing the abbreviation entirely as lowercase. For class definitions, its initial letter should be capitalized.

https://github.com/WordPress/gutenberg/blob/master/docs/reference/coding-guidelines.md#abbreviations-and-acronyms